### DISCUSSION TITLE
N64 RetroArch menu glitch makes it impossible to use

### DISCUSSION THREAD

**Discussion by milktoastrat** [rg35xx-cfw/rg35xx-cfw.github.io#213] [Q&A]
So far the N64 is the only system that I am seeing this issue. I have tried many different ways of fixing it but have had no luck this is apparently an issue for everyone with this release installed. When I use the hotkey while using libretro mupen64plus, the retroarch menu appears but its just a strange gradient with no text. You can see it move when you press up and down but its impossible to see what you are doing. Would like to see a fix for this so I can tinker a bit more with N64 emulation settings. THANK YOU
![WhatsApp Image 2024-04-03 at 10 34 08_5ad9f24d](https://github.com/rg35xx-cfw/rg35xx-cfw.github.io/assets/158623885/f66aaa88-39ac-461e-a12d-a72897fbe0ca)

**Comment by mcfizzlestag** [comment]
have you tried the fix outlined in:
https://github.com/rg35xx-cfw/rg35xx-cfw.github.io/issues/197
